eppo/src/main/java/cloud/eppo/android/EppoHttpClient.java

```diff
@@ -19,17 +27,23 @@
 public class EppoHttpClient {
     private static final String TAG = logTag(EppoHttpClient.class);
 
-    private final OkHttpClient client = new OkHttpClient().newBuilder()
-            .connectTimeout(10, TimeUnit.SECONDS)
-            .readTimeout(10, TimeUnit.SECONDS)
-            .build();
+    private final OkHttpClient client;
 
     private final String baseUrl;
     private final String apiKey;
 
     public EppoHttpClient(String baseUrl, String apiKey) {
         this.baseUrl = baseUrl;
         this.apiKey = apiKey;
+        this.client = buildOkHttpClient();
+    }
+
+    private static OkHttpClient buildOkHttpClient() {
+        OkHttpClient.Builder builder = new OkHttpClient().newBuilder()
+                .connectTimeout(10, TimeUnit.SECONDS)
+                .readTimeout(10, TimeUnit.SECONDS);
+
+        return builder.build();
```

Comment:
Made this more obvious it was happening in constructor and set it up for dropping in IpV4-only DNS, which may be needed (but hopefully not 🤞)